src/services/jwt.js

```diff
@@ -0,0 +1,41 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '5s',
```

Comment:
The expiration time for the JWT is set to '5s', which is quite short for most applications. Consider increasing the expiration time to a more reasonable duration unless this is intentional for testing purposes.